workflow/executor/docker/docker.go

```diff
@@ -72,13 +73,20 @@ func (d *DockerExecutor) CopyFile(containerID string, sourcePath string, destPat
 func (d *DockerExecutor) GetOutputStream(containerID string, combinedOutput bool) (io.ReadCloser, error) {
 	cmd := exec.Command("docker", "logs", containerID)
 	log.Info(cmd.Args)
-	if combinedOutput {
-		cmd.Stderr = cmd.Stdout
-	}
+
 	reader, err := cmd.StdoutPipe()
 	if err != nil {
 		return nil, errors.InternalWrapError(err)
 	}
+
+	if combinedOutput {
+		stderr, err := cmd.StderrPipe()
+		if err != nil {
+			return nil, errors.InternalWrapError(err)
+		}
+		reader = ioutil.NopCloser(io.MultiReader(reader, stderr))
```

Comment:
maybe we do not close this correctly

Reply:
I'm not sure it needs to though. We aren't going to leak file handles as the executor actually exits after this. I've not seen any problems.